code/modules/mob/living/carbon/xenomorph/castes/puppet/abilities_puppet.dm

```diff
@@ -19,7 +19,7 @@
 	HEAL_XENO_DAMAGE(owner_xeno, amount, FALSE)
 	add_cooldown()
 
-/datum/action/xeno_action/activable/flay/can_use_ability(mob/living/target, silent = FALSE, override_flags)
+/datum/action/xeno_action/activable/feed/can_use_ability(mob/living/target, silent = FALSE, override_flags)
```

Comment:
This feels like its a bugfix and should be noted? (unless its duplicate code again)

Reply:
it's unused code, puppets aren't controlled by players and their ai_behavior never calls this